src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,10 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        UserService service = new UserService();
```

Comment:
It is better to move this line to the class level. Becouse otherwise a new object of the type UserService will be created each time this method is called

Reply:
done